include/swift/AST/Attr.h

```diff
@@ -1329,6 +1333,28 @@ class ReferenceOwnershipAttr : public DeclAttribute {
   }
 };
 
+/// Represents an actorIndependent/actorIndependent(unsafe) decl attribute.
+class ActorIndependentAttr : public DeclAttribute {
+public:
+  ActorIndependentAttr(SourceLoc atLoc, SourceRange range, ActorIndependentKind kind)
+      : DeclAttribute(DAK_ActorIndependent, atLoc, range, /*Implicit=*/false) {
+    Bits.ActorIndependentAttr.kind = unsigned(kind);
+  }
+
+  ActorIndependentAttr(ActorIndependentKind kind)
```

Comment:
You can remove this constructor by adding an `= false` to the `isImplicit` parameter of the subsequent constructor.